build/make-initrd.sh

```diff
@@ -14,6 +14,7 @@ mkdir -m 0755 -p /tmp/hyperstart-rootfs/dev \
 cp ../src/init /tmp/hyperstart-rootfs
 cp busybox /tmp/hyperstart-rootfs
 cp iptables /tmp/hyperstart-rootfs
+cp ipvsadm /tmp/hyperstart-rootfs
```

Comment:
could you please directly copy all the above binaries into /tmp/hyperstart-rootfs/sbin?